code/modules/cm_aliens/XenoStructures.dm

```diff
@@ -492,7 +492,7 @@
 	if(!istype(M))
 		return attack_hand(M)
 
-	if(M.hivenumber != hivenumber)
+	if(!issamexenohive(M))
```

Comment:
We should really decide on what case to use when that happen : `is_same_xeno_hive()` or `isSameXenoHive()` or ...

Reply:
This is not something we currently enforce, though I'm a fan of `is_same_xeno_hive()` for clarity.
I had to stop and pay careful attention to see what the heck `issamexenohive()` was about.

Reply:
issamexenohive follows the format of the other isprocs like ishuman etc

Reply:
^

Reply:
Sad but true.